fixtures/cineroid/fl800.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "FL800",
+  "categories": ["Dimmer"],
+  "meta": {
+    "authors": ["Dyami Caliri"],
+    "createDate": "2024-02-05",
+    "lastModifyDate": "2024-02-05"
+  },
+  "links": {
+    "manual": [
+      "http://www.cineroid.com/img/item/ML_FL_Manaul_ENG%5B4%5D.pdf"
```

Comment:
Since the FL800 has the same user manual (it's for "FL400 / FL800 / ML800"), and both LED panels appear to use exactly the same controller, I wouldn't be opposed to combining these into one OFL file.  Only the size/weight/power are different.